src/autoComplete/AutoComplete.story.mdx

```diff
@@ -55,10 +56,10 @@ To display default items gives a list of `AutocompleteItem` to the `renderEmptyS
 
 <Canvas>
   <Story name="With empty search">
-    <BaseSection>
-      <p>
+    <BaseSection noHorizontalSpacing>
```

Comment:
Please remove `BaseSection` from `AutoComplete` stories, we can't have the props on canvas :(